### ISSUE TITLE
wallet transfer

### ISSUE THREAD

**Issue by ravi19821982** [AlphaWallet/alpha-wallet-android#3120]
hello i have more then 10 wallet  created i want to transfer all wallet in metamask how can i do that

**Comment by justindg** [comment]
> hello i have more then 10 wallet created i want to transfer all wallet in metamask how can i do that

Hi @ravi19821982 , go to `Settings` > `Show Seed Phrase` and use it to import your wallet using Metamask.

**Comment by ravi19821982** [comment]
but we can retrive only 1 account by seed phrase  i have 5 account

**Comment by JamesSmartCell** [comment]
> but we can retrive only 1 account by seed phrase i have 5 account

Can you change accounts and view the seed phrase for each?

**Comment by ravi19821982** [comment]
@JamesSmartCell  yes i can change account and i can  view seed phrase  but there is no option to see wallet private key in  alpha wallet ??

**Comment by JamesSmartCell** [comment]
> @JamesSmartCell yes i can change account and i can view seed phrase but there is no option to see wallet private key in alpha wallet ??

Ahh I see, you want to be able to dump the raw private key rather than seed phrase. Because MetaMask only uses one seed phrase and generates new keys using a modulus from that. 

We'll have a chat internally about this.
